src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -448,6 +448,47 @@ def setUp(self):
         self.d_log = DaosLog(self.context)
         self.test_log.daos_log = self.d_log
 
+        # Add test binaries and daos binaries to PATH
+        test_dirs = {"TESTING": "tests", "install": "bin"}
+        found_path = False
+        path_dirname = os.path.dirname(os.path.abspath(__file__))
+
+        # Use the developer environment from which this python file was called
+        if ("DAOS_ENV" in os.environ) and (os.environ["DAOS_ENV"] == "dev"):
```

Comment:
So Alexander may execute the test with no failure but Ethan execution of it could fail; not due to problems in the code or test but just because their login environment is slightly different.

Reply:
Yes, certain issues might only manifest in a production (RPM) environment versus developer installation tree.  This change provides a way to switch between the two.

Reply:
As long as this issue is recognized and understood.